<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="sunilr201-PC" failures="1" tests="1" name="Testdemo" time="0.041" errors="0" timestamp="2024-11-29T20:46:31 IST">
  <testcase classname="utilities.DataProviders" name="dataTest" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NullPointerException">
      <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.lang.NullPointerException
at java.base/java.io.File.<init>(File.java:278)
at utilities.DataProviders.dataSupplier(DataProviders.java:30)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- dataTest -->
</testsuite> <!-- Testdemo -->
